src/js/server/apply/router.js

```diff
@@ -275,4 +280,17 @@ function checkHasApplied(req, res, next) {
   }).catch(next);
 }
 
+/**
+ * Intercepts requests to check if applications are still open, redirecting to the dashbaord if not
+ */
+function checkApplicationsOpen(req, res, next) {
+  console.log(process.env.APPLICATIONS_OPEN);
+  if (process.env.APPLICATIONS_OPEN_STATUS === 'closed') {
```

Comment:
It might be overkill, but I tend to move environment variables to an exported function or constant. Referring to environment variables is kind of like referring to hard-coded strings.

That's if we actually had a type system 😬